code/datums/weakref.dm

```diff
@@ -1,21 +1,17 @@
-/datum
-	var/datum/weakref/weakref
-
-/datum/weakref
+/weakref

```

Comment:
I'd rather we not go to implicit datum declarations. It obfuscates code and inheritance.